### PR TITLE
WEBAPP-565: Fix version in debug builds

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/). 
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **WEBAPP**.

### CODE REVIEW

File: build/webpack.config.babel.js [hunk 1]
```diff
@@ -5,10 +5,17 @@ const HtmlWebpackPlugin = require('html-webpack-plugin')
...
+const SHORT_COMMIT_SHA_LENGTH = 8
+
+const readVersionName = () => {
+  const versionFile = fs.readFileSync(path.resolve(__dirname, '../version.json'))
+  return JSON.parse(versionFile).versionName
+}
 
 const createConfig = (env = {}) => {
-  const { config_name: buildConfigName, production } = env
+  const { config_name: buildConfigName, production, commit_sha: commitSha, version_name: versionName } = env
```
Comment (klinzo) [inline build/webpack.config.babel.js:18]: Will be used in https://issues.integreat-app.de/browse/WEBAPP-536

**Comment by klinzo** [comment]
@maxammann any idea why the check fails? I don't now I could debug it.

File: .circleci/config.yml [hunk 2]
```diff
@@ -192,7 +192,9 @@ jobs:
           steps:
             - run:
                 name: Read version number
-                command: echo 'export VERSION_NAME=$(cat version.json | jq .versionName)' >> $BASH_ENV
+                command: |
+                  echo "export VERSION_NAME=$(cat version.json | jq .versionName)" >> $BASH_ENV
+                  source $BASH_ENV
```
Comment (maxammann) [inline .circleci/config.yml:197]: This line can be removed. The BASH_ENV is sourced in each step

File: build/webpack.config.babel.js [hunk 3]
```diff
@@ -5,10 +5,17 @@ const HtmlWebpackPlugin = require('html-webpack-plugin')
...
+const SHORT_COMMIT_SHA_LENGTH = 8
+
+const readVersionName = () => {
+  const versionFile = fs.readFileSync(path.resolve(__dirname, '../version.json'))
+  return JSON.parse(versionFile).versionName
+}
 
 const createConfig = (env = {}) => {
-  const { config_name: buildConfigName, production, debug } = env
+  const { config_name: buildConfigName, production, debug, commit_sha: commitSha, version_name: versionName } = env
```
Comment (maxammann) [inline build/webpack.config.babel.js:18]: Where is `commitSha` coming from?

Comment (klinzo) [reply build/webpack.config.babel.js:18]: Supplied by CircleCI. It is not necessary though.

File: .circleci/config.yml (hunk 2 above)
Comment (klinzo) [reply .circleci/config.yml:197]: Will be removed in https://github.com/Integreat/integreat-webapp/pull/352

**Pull request merged**